apps/desktop-wallet/package.json

```diff
@@ -4,22 +4,32 @@
   "version": "2.1.5",
   "author": "Alephium dev <dev@alephium.org>",
   "main": "public/electron.js",
+  "homepage": "./",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:alephium/alephium-frontend.git"
+  },
```

Comment:
`homepage` and `repository` are needed for `electron-builder` to work.

`private` was added since the package is not published on npm.